services/galley/src/Galley/API/Update.hs

```diff
@@ -282,11 +279,13 @@ uncheckedUpdateConversationAccess body usr zcon conv (currentAccess, targetAcces
   let removedUsers = map lmId users \\ map lmId newUsers
       removedBots = map botMemId bots \\ map botMemId newBots
   mapM_ (deleteBot cnv) removedBots
-  case removedUsers of
-    [] -> return ()
-    x : xs -> do
-      -- FUTUREWORK: deal with remote members, too, see removeMembers (Jira SQCORE-903)
-      e <- Data.removeLocalMembersFromLocalConv localDomain conv (Qualified usr localDomain) (x :| xs)
+  case nonEmpty removedUsers of
+    Nothing -> return ()
+    Just victims -> do
```

Comment:
```suggestion
  for (nonEmpty removedUsers) $ \victims -> do
```

Reply:
Thanks, that's a nicer. 04785b4c17c9b1cbcd53cf060c0bbd3cb150aa23.